src/main/resources/messages/codegpt.properties

```diff
@@ -4,7 +4,7 @@ action.generateCommitMessage.title=Generate Message
 action.generateCommitMessage.description=Generate commit message
 action.generateCommitMessage.serviceWarning=Messages can only be generated with OpenAI or Azure service
 action.generateCommitMessage.missingCredentials=Credentials not provided
-action.includeFilesInContext.title=Include In Context...
+action.includeFilesInContext.title=Include File In Context...
```

Comment:
I added 'File' here to make it clear to the user if he right clicks in the Editor it adds the File, not the text at the cursor or similar.

Reply:
Perhaps it's better to have two separate keys, because otherwise the label will be wrong when you want to select a folder of files.